### DISCUSSION TITLE
Como transferir fundos

### DISCUSSION THREAD

**Discussion by Johnni171** [btcpayserver/btcpayserver#5215] [General]
Como transferir meus biticons para binance

**Comment by Skyeenicole** [comment]
napkin faculty van young say provide piano seek october bean power rudeOn Jul 30, 2023, at 6:05 PM, Johnni171 ***@***.***> wrote:﻿
Como transferir meus biticons para binance

**Comment by pavlenex** [comment]
It's a scam that has nothing to do with our software https://blog.btcpayserver.org/bitcoin-bonus-scam/
